docs/dev/log_integration.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Create a Log Integration
+kind: documentation
+aliases:
+- /logs/faq/partner_log_integration
+further_reading:
```

Comment:
Can we add a link to the Log Facets doc since it was taken out of the main doc? https://docs.datadoghq.com/logs/explorer/facets/